polyfill/test/plaintime.mjs

```diff
@@ -18,214 +18,15 @@ import * as Temporal from 'proposal-temporal';
-      it('time.second is 30', () => equal(time.second, 30));
-      it('time.millisecond is 123', () => equal(time.millisecond, 123));
-      it('time.microsecond is 456', () => equal(time.microsecond, 456));
-      it('time.nanosecond is 0', () => equal(time.nanosecond, 0));
-      it('`${time}` is 15:23:30.123456', () => equal(`${time}`, '15:23:30.123456'));
-    });
-    describe('missing microsecond', () => {
-      let time;
-      it('time can be constructed', () => {
-        time = new PlainTime(15, 23, 30, 123);
-        assert(time);
-        equal(typeof time, 'object');
-      });
-      it('time.hour is 15', () => equal(time.hour, 15));
-      it('time.minute is 23', () => equal(time.minute, 23));
-      it('time.second is 30', () => equal(time.second, 30));
-      it('time.millisecond is 123', () => equal(time.millisecond, 123));
-      it('time.microsecond is 0', () => equal(time.microsecond, 0));
-      it('time.nanosecond is 0', () => equal(time.nanosecond, 0));
-      it('`${time}` is 15:23:30.123', () => equal(`${time}`, '15:23:30.123'));
-    });
-    describe('missing millisecond', () => {
-      let time;
-      it('time can be constructed', () => {
-        time = new PlainTime(15, 23, 30);
-        assert(time);
-        equal(typeof time, 'object');
-      });
-      it('time.hour is 15', () => equal(time.hour, 15));
-      it('time.minute is 23', () => equal(time.minute, 23));
-      it('time.second is 30', () => equal(time.second, 30));
-      it('time.millisecond is 0', () => equal(time.millisecond, 0));
-      it('time.microsecond is 0', () => equal(time.microsecond, 0));
-      it('time.nanosecond is 0', () => equal(time.nanosecond, 0));
-      it('`${time}` is 15:23:30', () => equal(`${time}`, '15:23:30'));
-    });
-    describe('missing second', () => {
-      let time;
-      it('time can be constructed', () => {
-        time = new PlainTime(15, 23);
-        assert(time);
-        equal(typeof time, 'object');
-      });
-      it('time.hour is 15', () => equal(time.hour, 15));
-      it('time.minute is 23', () => equal(time.minute, 23));
-      it('time.second is 0', () => equal(time.second, 0));
-      it('time.millisecond is 0', () => equal(time.millisecond, 0));
-      it('time.microsecond is 0', () => equal(time.microsecond, 0));
-      it('time.nanosecond is 0', () => equal(time.nanosecond, 0));
-      it('`${time}` is 15:23:00', () => equal(`${time}`, '15:23:00'));
-    });
-    describe('missing minute', () => {
-      const time = new PlainTime(15);
-      it('`${time}` is 15:00:00', () => equal(`${time}`, '15:00:00'));
-    });
-    describe('missing all parameters', () => {
-      const time = new PlainTime();
-      it('`${time}` is 00:00:00', () => equal(`${time}`, '00:00:00'));
-    });
-  });
   describe('.with manipulation', () => {
     const time = new PlainTime(15, 23, 30, 123, 456, 789);
-    it('time.with({ hour: 3 } works', () => {
```

Comment:
These are covered by PlainTime/prototype/with/order-of-operations.js but it could be worth adding another test file that does each property individually like these tests do here?